gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -19,6 +19,7 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
+    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
```

Comment:
if i understand the maven conventions, if you check it in, then `src/*` (and i would vote for `src/generated/*`), and if you don't it should go in `build/*` (the gradle equivalent of `target/`)